src/network/tcp.py

```diff
@@ -93,12 +95,14 @@ def antiIntersectionDelay(self, initial = False):
                 self.skipUntil = time.time() + delay
 
     def state_connection_fully_established(self):
+        """TBC"""
```

Comment:
State after the bitmessage protocol handshake is completed (version/verack exchange, and if both side support TLS, the TLS handshake as well).